tests/e2e/core-tests/specs/merchant/wp-admin-product-import-csv.test.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable jest/no-export, jest/no-disabled-tests */
+/**
+ * Internal dependencies
+ */
+ const {
+	merchant,
+	setCheckbox,
+	moveAllItemsToTrash
+} = require( '@woocommerce/e2e-utils' );
+
+/**
+ * External dependencies
+ */
+const {
+	it,
+	describe
+} = require( '@jest/globals' );
+
+const filePath = '../../../sample-data/sample_products.csv';
+const filePathOverride = '../../../tests/e2e/core-tests/test-data/sample_products_override.csv';
```

Comment:
This path won't work in the package. When another project uses the package it is installed in `node_modules/@woocommerce/e2e-core-tests`. They won't have a `core-tests` folder. https://github.com/woocommerce/woocommerce/blob/trunk/tests/e2e/env/utils/app-root.js has the logic you can use to determine which folder path to use. You could make a `getCoreTestRoot()` in `core-tests` which might be re-used in the future.

Reply:
Thanks, @rrennick I did that way!